docs/api/hardhat/plugins.md

```diff
@@ -411,3 +411,98 @@ expect(await contract.getAmount()).to.equal(100);
+
+This plugin is used to verify contracts on the zkSync 2.0 network.
+
+### Npm
+
+[@matterlabs/hardhat-zksync-verify](https://www.npmjs.com/package/@matterlabs/hardhat-zksync-verify)
+
+This plugin is used in conjunction with [@nomiclabs/hardhat-etherscan](https://www.npmjs.com/package/@nomiclabs/hardhat-etherscan) and it supports backward compatibility with that plugin.
+To verify contracts on zkSync, you have to install and import `@matterlabs/hardhat-zksync-verify` in the `hardhat.config.ts` file.
+
+```javascript
+import "@matterlabs/hardhat-zksync-verify";
+```
+
+Add the latest version of this plugin to your project with the following command:
+
+``` typescript
+# Yarn
+yarn add -D @matterlabs/hardhat-zksync-verify
+
+# Npm
+npm i -D @matterlabs/hardhat-zksync-verify
+```
+
+### Exports
+
+### Configuration
+
+```typescript
+networks: {
+  goerli: {
+    url: "https://goerli.infura.io/v3/<API_KEY>" // URL of the Ethereum Web3 RPC (optional)
+  },
+  zkTestnet: {
+    url: "https://zksync2-testnet.zksync.dev", // URL of the zkSync network RPC
+    ethNetwork: "goerli", // URL of the Ethereum Web3 RPC, or the identifier of the network (e.g. `mainnet` or `goerli`)
+    zksync: true,
+    verifyURL: 'https://zksync2-testnet-explorer.zksync.dev/contract_verification'
+  }
+},
+// defaultNetwork: "zkTestnet", // optional (if not set, use '--network zkTestnet')
+```
+
+- `zkTestnet` is an arbitrary zkSync network name. You can select this as the default network using the `defaultNetwork` property.
+- `ethNetwork` is a field with the URL of the Ethereum node. You can also provide the network name (e.g. `goerli`) as the value of this field. In this case, the plugin will either use the URL of the appropriate Ethereum network configuration (from the `networks` section), or the default `ethers` provider for the network if the configuration is not provided. This field is required for all zkSync networks used by this plugin.
+- `url` is a field with the URL of the zkSync node in the case of the zkSync network (with `zksync` flag set to `true`), or the URL of the Ethereum node. This field is required for all zkSync and Ethereum networks used by this plugin.
+- `zksync` is a flag to indicate if the network represents zkSync network configuration. This field needs to be set to `true` for all zkSync networks. If you want to run a `hardhat-etherscan` verification, this field needs to be set to `false`. If set to `true`, the verification process will always try to run the verification process on the zkSync network.
+- `verifyURL` is a field that points to the verification endpoint for the specific zkSync network. This parameter is optional, and its default value is the testnet verification url (`https://zksync2-testnet-explorer.zksync.dev/contract_verification`). In a case of a need to use a different url, aside from changing the parameter, a user also has to add `import "./type-extensions.ts` command to the `index.ts` file in the verify plugin.
```

Comment:
This is now invalid right? The import part I mean @FilipTxFusion

Reply:
Yes, it is not needed for the next release. I updated that part.